tests/test_utils.py

```diff
@@ -70,3 +101,56 @@ def get_test_job(provider: ProviderName) -> JobRequestDto:
     elif provider == ProviderName.AWS:
         job_dict: dict = get_object_from_json(JOB_JSON_AWS)
         return JobRequestDto(**job_dict)
+
+
+def check_simple_job_dto(return_dto: SimpleJobDto):
+    assert return_dto.id == EXPECTED_ID
+    assert return_dto.name == STANDARD_JOB_NAME
+    assert return_dto.state == JobState.RUNNING
+
+
+def check_if_job_finished(job: JobDataclass):
+    assert job.id == EXPECTED_ID
+    assert job.progress == JOB_FINISHED_PROGRESS
+    assert job.state == JobState.FINISHED
+
+
+def ibm_check_if_job_runner_result_correct(job: JobDataclass):
+    job.type = JobType.RUNNER
+    for i in range(len(job.results)):
+        result: ResultDataclass = job.results[i]
+        check_standard_result_data(i, job, result)
+        assert result.meta_data is not None
+        shots: int = job.shots
+        if i == 0:
+            assert (shots / 2 + RESULT_TOLERANCE) > result.result_dict["0x0"] > (shots / 2 - RESULT_TOLERANCE)
+            assert (shots / 2 + RESULT_TOLERANCE) > result.result_dict["0x3"] > (shots / 2 - RESULT_TOLERANCE)
+            assert (result.result_dict["0x0"] + result.result_dict["0x3"]) == shots
+        else:
+            assert result.result_dict["0x0"] == shots
+
+
+def check_aws_local_simulator_results(results, shots: int):
+    for i in range(len(results)):
+        results_dict = results[i].result_dict
+        counts: Counter = results_dict.get("counts")
+        probabilities: dict = results_dict.get("probabilities")
+        if i == 0:
+            if counts.get("00") is not None and counts.get("11") is not None:
+                counts0 = counts.get("00")
+                probabilities0 = probabilities.get("00")
```

Comment:
denke das wäre ein probability_first oder so schöner. Ist auch ein Wert also singular

Reply:
Hier noch ein Vorschlag: probability00 und probabilty11, aber alles fine for me